README.md

```diff
@@ -1,126 +1,249 @@
+
+Below are some LMS screenshots of a mentoring block in assessment mode.
+
+Question before submitting an answer:
+
+![Assessment Step 1](doc/img/assessment-1.png)
+
+Question after submitting the correct answer:
+
+![Assessment Step 2](doc/img/assessment-2.png)
+
+Question after submitting a wrong answer:
+
+![Assessment Step 3](doc/img/assessment-3.png)
+
+Score review and the "Try Again" button:
+
+![Assessment Step 4](doc/img/assessment-4.png)
+
+### Free-form questions
+
+Free-form questions are represented by a "Long Answer" component. 
+
+Example screenshot before answering the question:
+
+![Answer Initial](doc/img/answer-1.png)
+
+Screenshot after answering the question:
+
+![Answer Complete](doc/img/answer-2.png)
+
+You can add "Long Answer Recap" components to mentoring blocks later on
+in the course to provide a read-only view of any answer that the student
+entered earlier.
+
+The read-only answer is rendered as a quote in the LMS:
+
+![Answer Read-Only](doc/img/answer-3.png)
+
+### Multiple Choice Questions
+
+Multiple choice questions can be added to a mentoring component and
+have the following configurable options:
+
+* Question - The question to ask the student
+* Message - A feedback message to display to the student after they
+  have made their choice.
+* Weight - The weight is used when computing total grade/score of
+  the mentoring block. The larger the weight, the more influence this
+  question will have on the grade. Value of zero means this question
+  has no influence on the grade (float, defaults to `1`).
+* Correct Choice - Specify which choice[s] is considered correct. If
+  a student selects a choice that is not indicated as correct here,
+  the student will get the question wrong.
+
+Using the Studio editor, you can add "Custom Choice" blocks to the MCQ.
+Each Custom Choice represents one of the options from which students
+will choose their answer.
+
+You can also add "Tip" entries. Each "Tip" must be configured to link
+it to one or more of the choices. If the student chooses a choice, the
+
+
+Screenshot: Before attempting to answer the questions:
+
+![MCQ Initial](doc/img/mcq-1.png)
+
+While attempting to complete the questions:
+
+![MCQ Attempting](doc/img/mcq-2.png)
+
+After successfully completing the questions:
+
+![MCQ Success](doc/img/mcq-3.png)
+
+#### Rating MCQ
+
+When constructing questions where the student rates some topic on the
+scale from `1` to `5` (e.g. a Likert Scale), you can use the Rating
+question type, which includes built-in numbered choices from 1 to 5
+The `Low` and `High` settings specify the text shown next to the
+lowest and highest valued choice.
+
+Rating questions are a specialized type of MCQ, and the same
+instructions apply. You can also still add "Custom Choice" components
+if you want additional choices to be available such as "I don't know".
+
+
+### Self-assessment MRQs
+
+Multiple response questions are set up similarly to MCQs. The answers
```

Comment:
Write out Multiple Response Questions, putting MRQ in parenthesis in the header here, then capitalize 'Response' and 'Questions' in this sentence.